rosdep/base.yaml

```diff
@@ -2481,6 +2485,8 @@ libqt5-widgets:
   slackware: [qt5]
   ubuntu: [libqt5widgets5]
 libqt5x11extras5-dev:
+  debian: [libqt5x11extras5-dev]
+  fedora: [libQt5X11Extras-devel]
```

Comment:
I think that we use only RPM available on the Fedora package database. Maybe @wjwwood or @rmattes can confirn this